velox/expression/tests/utils/ArgumentTypeFuzzer.h

```diff
@@ -69,6 +69,8 @@ class ArgumentTypeFuzzer {
   /// randomly generated type.
   void determineUnboundedTypeVariables();
 
+  void determineUnboundedIntegerVariables(const exec::TypeSignature& type);
```

Comment:
nit: it seems only decimal type is supported. Do you need to document this in the comment?

Reply:
Good point. Added comments:

```
  // Bind integer variables used in specified decimal(p,s) type signature to
  // randomly generated values if not already bound.
  // Noop if 'type' is not a decimal type signature.
  void determineUnboundedIntegerVariables(const exec::TypeSignature& type);
```